etc/schema_hideme.txt

```diff
@@ -0,0 +1,29 @@
+# Properties to hide from human readable schema documentation
```

Comment:
The main problem I see here is that _somebody_ is going to go look at the JSON specification anyway, which will then be inconsistent with the generated documentation. I'm not sure there's any great solution here, but just something to keep in mind.

The real problem is that people shouldn't be _coding to the schema_ since it's a very incomplete specification compared to what the behavior aspects of the system are. 

Nit: The other way to handle this would be to have something in the schema itself (e.g. "$when") that indicates when it's required/allowed -- so at least all the code/specification is in one place, rather than spread across different files.  I.e. consider having this information pulled from the schema itself, not from this file.